homeassistant/components/google/calendar.py

```diff
@@ -174,13 +201,13 @@ async def async_get_events_page(self, hass, events, params, page_token, event_li
                     event_list.append(item)
             else:
                 event_list.append(item)
-        return result.get("nextPageToken")
+        return cast(str, result.get("nextPageToken"))
```

Comment:
We can't cast to string and use dict.get at the same time. `dict.get` may return `None`.